submit-web/src/hooks/api/useProjects.tsx

```diff
@@ -39,10 +39,26 @@ export const useLoadProjectsByProponentId = (proponentId?: number) => {
 
 type GetProjectsByAccountParams = {
   accountId?: number;
+  searchOptions?: Record<string, string | number>;
 };
-const getProjectsByAccount = ({ accountId }: GetProjectsByAccountParams) => {
+
+const getProjectsByAccount = ({
+  accountId,
+  searchOptions,
+}: GetProjectsByAccountParams) => {
+  // Initialize URL with base path and account ID
+  let url = `/projects/accounts/${accountId}`;
```

Comment:
I don't think you have to manually pass the args to the url like this, I'm sure there's a params prop or something in axios with which u can pass these params  and axios will handle it